tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -314,6 +314,7 @@ def test_create_model_version_with_langchain_dependencies(store, langchain_local
             secret_access_key=secret_access_key,
             session_token=session_token,
             credential_refresh_def=ANY,
+            s3_upload_extra_args={},
```

Comment:
Any way to add a test/mock around this not being empty every time?

Reply:
Added tests in https://github.com/mlflow/mlflow/pull/12495/commits/d8bd736bd1d02997aa04a436d5067a7aa739022d